source/github/webhook.go

```diff
@@ -319,12 +320,23 @@ func processIssueCommentEvent(h *library.Hook, payload *github.IssueCommentEvent
 	b.SetSender(payload.GetSender().GetLogin())
 	b.SetAuthor(payload.GetIssue().GetUser().GetLogin())
 	b.SetEmail(payload.GetIssue().GetUser().GetEmail())
-	b.SetRef(fmt.Sprintf("refs/pull/%d/head", payload.GetIssue().GetNumber()))
+	// treat as non-pull-request comment by default and
+	// set ref to default branch for the repo
+	b.SetRef(fmt.Sprintf("refs/heads/%s", r.GetBranch()))
```

Comment:
Open to other options here. Pull request ref doesn't make sense for normal issue comments, though.

Reply:
I think what you have makes sense 👍 

Ideally, if they are commenting on a normal issue, we should inherit the "default" branch from the repo.

The only thing I can think of is maybe in a future iteration we allow them to provide a ref/branch/commit etc via some specially formatted comment? 🤷‍♂️ 

But that's way beyond the scope of what you have here!